src/meepgeom.cpp

```diff
@@ -2567,4 +2567,26 @@ void material_grids_addgradient(meep::realnum *v, size_t ng, std::complex<double
   }
 }
 
+void get_epsilon_grid(geometric_object_list gobj_list,
+                      material_type_list mlist,
+                      material_type _default_material,
+                      bool _ensure_periodicity,
+                      meep::grid_volume gv,
+                      vector3 cell_size,
+                      vector3 cell_center,
+                      int nx, double *x,
+                      int ny, double *y,
+                      int nz, double *z,
```

Comment:
`x,y,z` should be `const double *` since they are not modified in this function

Reply:
Fixed.